site/docs/voice/about.mdx

```diff
@@ -60,3 +98,11 @@ The Voice API allows you to playback audio files to end users on phone calls. On
 ### Recordings
 
 Bandwidth will keep recordings for up to 30 days. After 30 days the recordings will be deleted from Bandwidth's servers.
+
+### Transcriptions
+
+Bandwidth will keep transcriptions for up to 30 days. After 30 days the transcriptions will be deleted from Bandwidth's servers.
```

Comment:
Nick's suggestion: `Bandwidth will keep transcriptions for up to 30 days. After 30 days, the transcriptions will be marked for deletion, which may take up to `48` hours to fully process.` but I feel like that's more info than the customers need and imo might encourage banking on having access after 30 days

Reply:
Yeah, I wouldn't disclose anything about how fast or slow we are to delete things. We promise 30, leave the rest off

Reply:
I agree; @nashley are you ok with leaving this as is?

Reply:
I'd prefer to be explicit -- especially since promising to delete customer data and then not actually deleting it until later (even if unintentionally) isn't great. It's the kind of thing I'd want to know as a customer.
However, that's just my personal opinion, and I don't think there's the expectation of us being 100% accurate, and we're probably not legally obligated to be? GDPR may change that though.